index.js

```diff
@@ -21,7 +22,12 @@ if (process.argv[2] && process.argv[2].toLowerCase() == 'b') {
         }).join(''));
       }
   })
-  .catch(function(){})
+  .catch(function(err){
+    if(err.statusCode === 404) {
+      process.stdout.write(sorryMsg);
+      sorryMsg = '.';
```

Comment:
So that it, keeps writing a `.` to the `stdout` instead of printing the entire message.